releng/org.eclipse.xtext.target/org.eclipse.xtext.target.target

```diff
@@ -2,35 +2,65 @@
 <?pde version="3.8"?>
 <target name="org.eclipse.xtext.helios.target" sequenceNumber="6">
 <locations>
-<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
-<unit id="org.eclipse.jdt.feature.group" version="0.0.0"/>
-<unit id="org.eclipse.platform.feature.group" version="0.0.0"/>
-<unit id="org.eclipse.pde.api.tools.ee.feature.feature.group" version="0.0.0"/>
-<unit id="org.eclipse.pde.feature.group" version="0.0.0"/>
-<unit id="org.eclipse.draw2d.feature.group" version="0.0.0"/>
-<unit id="com.google.guava" version="0.0.0"/>
-
-<unit id="org.eclipse.emf.sdk.feature.group" version="0.0.0"/>
-
-<unit id="org.eclipse.xpand" version="0.0.0"/>
-<unit id="org.eclipse.xtend" version="0.0.0"/>
-<unit id="org.eclipse.xtend.typesystem.emf" version="0.0.0"/>
-<unit id="org.eclipse.emf.mwe2.lib" version="0.0.0"/>
-
-<unit id="org.eclipse.m2e.feature.feature.group" version="1.5.1.20150109-1820"/>
-
-<repository location="http://download.eclipse.org/releases/luna/201502271000/"/>
-</location>
-
-<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
-<unit id="org.eclipse.emf.mwe2.launcher.feature.group" version="0.0.0"/>
-<repository location="http://download.eclipse.org/modeling/emft/mwe/updates/releases/2.7.1/"/>
-</location>
-
-<location includeAllPlatforms="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-<unit id="de.itemis.xtext.antlr.sdk.feature.group" version="0.0.0"/>
-<repository location="http://download.itemis.com/updates/releases/2.1.1/"/>
-</location>
-
+	<location includeAllPlatforms="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+		<unit id="org.eclipse.xtext.xbase.lib" version="0.0.0"/>
+		<unit id="org.eclipse.xtend.lib" version="0.0.0"/>
+		<unit id="org.eclipse.xtend.lib.macro" version="0.0.0"/>
+		<repository location="http://services.typefox.io/open-source/jenkins/job/xtext-lib/job/msp_p2repo/lastStableBuild/artifact/build/p2-repository/"/>
```

Comment:
Yes, the master branch doesn't have any p2 repo yet. I will change this before merging.